test/api.js

```diff
@@ -750,7 +757,7 @@ describe('/v1', function() {
           });
         }
 
-        it('should get auth token with secret', function(done){
+        it('should get auth token with secret', function(){
           return getCode(clientId).then(function(code) {
```

Comment:
I'm a bit surprised there weren't problems with this test earlier since both a promise are returned and `done` specified.